arches/app/utils/label_based_graph_v2.py

```diff
@@ -0,0 +1,268 @@
+from arches.app.datatypes.datatypes import DataTypeFactory
```

Comment:
The only difference between this and V1 is the changing the `get_display_value` call to `to_json`, yeah? Should we be duplicating everything, or should we work versioning logic into the current endpoint? Or maybe follow an inheritance pattern and overwrite the one method?